triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/AuthServiceClient.scala

```diff
@@ -180,6 +180,20 @@ class AuthServiceClient(authServiceBaseUri: Uri)(
     )
     runRequest(req)(Unmarshal(_).to[LedgerAccessToken])
   }
+
+  def getLedgerToken(
```

Comment:
It looks like you only call this when starting a trigger. How do we validate credentials when stopping a trigger?

Reply:
Yes, will need to implement that next